addons/test_mail/static/tests/activity_tests.js

```diff
@@ -658,6 +659,104 @@ QUnit.module("test_mail", {}, function () {
         }
     );
 
+    QUnit.test(
+        "activity view: 'scheduleActivity' does not add activity_ids condition as selectCreateDialog domain",
+        async function (assert) {
+            patchWithCleanup(ActivityController.prototype, {
+                scheduleActivity() {
+                    this._super();
+                    assert.step(JSON.stringify(this.props.domain));
+                },
+            });
+
+            const { env, openView } = await start({
+                serverData,
+            });
+            await openView({
+                res_model: "mail.test.activity",
+                views: [[false, "activity"]],
+            });
+            patchWithCleanup(env.services.action, {
+                doAction(action, options) {
+                    assert.step("doAction");
+                    options.onClose();
+                },
+            });
```

Comment:
Hi, this is the redundant part in all 3 test cases. The patch above it addresses three different methods in 3 cases.
Also, I found the highlighted part in another existing test case as well. Do you want me to create a helper and make changes wherever applicable, or do I create a seperate module (checking activity_ids condition is not added) for these 3 test cases and add the common code in its beforeEach()?

Reply:
only for your change :) (we are in stable)